src/EditorFeatures/Test/Workspaces/TextFactoryTests.cs

```diff
@@ -73,7 +72,7 @@ public void TestCreateFromTemporaryStorage()
 
                 Assert.NotSame(text, text2);
                 Assert.Equal(text.ToString(), text2.ToString());
-                Assert.Equal(text2.Encoding, Encoding.Unicode);
+                Assert.Equal(text2.Encoding, null);
```

Comment:
Let's add another test (or enhance this one) to test non-null encodings too.

    foreach(var encoding in new Encoding[] {
        null,
        Encoding.ASCII,
        Encoding.UTF8,
        Encoding.Unicode,
        Encoding.BigEndianUnicode})
    {
        var text = Text.SourceText.From("Hello, World!", encoding);
        // etc.